client/src/components/Ticketing/GroupeToggle.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+
+const GroupToggle = () => {
```

Comment:
Since this is specifically for toggling between activeness for ticketing things, I'd be a little more specific with the component name rather than a generic GroupToggle, and then make the filename match

Reply:
fixed